StortSpelProjekt/Game/src/Gamefiles/Components/MeleeComponent.cpp

```diff
@@ -23,6 +24,10 @@ component::MeleeComponent::MeleeComponent(Entity* parent) : Component(parent)
 	// Fetch the player transform
 	m_pMeleeTransform = parent->GetComponent<component::TransformComponent>()->GetTransform();
 
+	// Fetch the player audio component
+	m_pVoiceComponent = parent->GetComponent<component::Audio2DVoiceComponent>();
+	m_pVoiceComponent->AddVoice(L"SwordSwing");
+
```

Comment:
Assuming that there will be a swordswing makes other scenes crash.
What would happen if an enemy has a meleecomponent and does not use this sound?